prototype_source/pt2e_quantizer.rst

```diff
@@ -302,6 +302,114 @@ functions that are used in the example:
    `get_bias_qspec <https://github.com/pytorch/pytorch/blob/47cfcf566ab76573452787335f10c9ca185752dc/torch/ao/quantization/_pt2e/quantizer/utils.py#L53>`__
    can be used to get the ``QuantizationSpec`` from ``QuantizationConfig`` for a specific pattern.
 
+A Note on IR for PT2E Quantization Flow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Who is the target audience for this note? Wouldn't I expect them to already know what an IR is?

Reply:
this is targeting backend developers, like ARM, Qualcomm, Intel, TensorRT etc. yeah they probably know what an IR is. you mean I can delete the explanation of IR or something? the explanation is also explaining what we mean by torch IR, aten IR etc.